Minor: update changelog to mention read-after-write consistency. (#2457)
confluentinc/ksql: docs/developer-guide/api.rst

          "commandStatus": {
            "status":"SUCCESS",
            "message":"Stream created and running"
          },
          "commandSequenceNumber":10
        },
        {
          "statementText":"CREATE STREAM pageviews_alice AS SELECT * FROM pageviews_original WHERE userid='alice';",
          "commandId":"stream/PAGEVIEWS_ALICE/create",
          "commandStatus": {
            "status":"SUCCESS",
            "message":"Stream created and running"
          },
          "commandSequenceNumber":11
        }
      ]

   .. _coordinate_multiple_requests

>    **Coordinating among multiple requests**

Reviewer: ```suggestion
   Coordinate Multiple Requests
   ---------------------------------
```

----------------------------

To submit multiple, interdependent requests, there are two options. The first is to submit them as a single request,

> similar to the example request above:

Reviewer 2: ```suggestion
similar to the example request below:
```

Author: Oh I actually meant above (see lines 201-214). Normally I'd throw in a link but it feels a little overkill since it really is the content directly above this sentence. If a link is preferred I can add it though.

Reviewer 2: oh... if it looks good compiled then you can ignore this

   .. code:: http

      POST /ksql HTTP/1.1
      Accept: application/vnd.ksql.v1+json
      Content-Type: application/vnd.ksql.v1+json

      {
        "ksql": "CREATE STREAM pageviews_home AS SELECT * FROM pageviews_original WHERE pageid='home'; CREATE TABLE pageviews_home_count AS SELECT userid, COUNT(*) FROM pageviews_home GROUP BY userid;"
      }

>    The second method is to submit the statements as separate requests and incorporate the interdependency via ``commandSequenceNumber``.

Reviewer: ```suggestion
   The second method is to submit the statements as separate requests and incorporate the interdependency by using ``commandSequenceNumber``.
```
